.github/workflows/dev-stg-release.yml

```diff
@@ -57,6 +57,7 @@ jobs:
 
       - name: Push to Staging
         if: github.event.inputs.bal_central_environment == 'STAGE'
+        working-directory: ./ballerina
```

Comment:
we can remove the ` WORKING_DIR` env variable with this

Reply:
Updated